resources/home/dnanexus/generate_workbook/tests/test_excel.py

```
@@ -0,0 +1,162 @@
+import os
+import sys
+import pytest
+from pathlib import Path
+from dxpy.bindings.dxfile_functions import open_dxfile
```

Ruff (0.11.9)

`dxpy.bindings.dxfile_functions.open_dxfile` imported but unused

Remove unused import: `dxpy.bindings.dxfile_functions.open_dxfile`

(F401)